cssApp.css

```diff
@@ -50,14 +53,57 @@ html, body {
     color: #555555;
 }
 
+/* backgrounds */
+
+#initial-background {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    max-height: calc(100% - 50px);
+}
+#world-map-background {
+    position: absolute;
+    max-height: 100%;
+    filter: blur(1.5px);
+}
+#headline {
+    animation: enlargeText 0.7s ease-out;
+    position: absolute;
+    top: 100px;
```

Comment:
move top property to where you declare left/right

Reply:
ok